app/controllers/api/LICENSE

```diff
@@ -0,0 +1,19 @@
+Copyright (c) 2012 Brian Ploetz (bploetz@gmail.com)
```

Comment:
Hmm I'm a bit confused now. My intuition right now says that I should just remove this license? Brian Ploetz is the founder of versionist. I'm actually now wondering where the license by Nov Matake is: taking a look now.